input/Glfw/GlfwInputDevice.cpp

```diff
@@ -10,9 +10,10 @@ namespace NovelRT::Input::Glfw
           _previousStates(std::vector<InputAction>()),
           _mousePos(NovelRT::Maths::GeoVector2F::Zero())
     {
+        _logger = NovelRT::LoggingService(NovelRT::Utilities::Misc::CONSOLE_LOG_INPUT);
     }
 
-    void GlfwInputDevice::Initialise(void* window)
+    void GlfwInputDevice::Initialise(NovelRT::Windowing::IWindowingDevice* device)
```

Comment:
Idk, for some reason it makes it easier for me to read, but functionally it doesn't matter since we're not caching it. (Plus smart ptrs are our convention as of now.) I've changed this over to be a `shared_ptr` instead.